### ISSUE TITLE
Option to "hide" the PR Comment

### ISSUE THREAD

**Issue by cwbusacker** [saurav0705/check-history-action#53]
Currently, it looks like the option is to only completely disable the PR comment. Nice to make it be hidden by default because it can take a lot of space.

**Comment by saurav0705** [comment]
@cwbusacker I agree will add this in the next release of the version and will update.

**Comment by cwbusacker** [comment]
@saurav0705, if you'd like, I wouldn't mind joining you with this open source project.

**Comment by saurav0705** [comment]
@cwbusacker sure any contribution is welcomed
you can fork the repo and submit the pr
